### DISCUSSION TITLE
Add Xbox One controller emulation support [nvm]

### DISCUSSION THREAD

**Discussion by encryptize** [LizardByte/community#20] [Sunshine] [answered]
~~**Host**: Sunshine 0.13.0 on Windows 10~~
~~**Client**: Moonlight 3.2.0 on Manjaro Linux with a connected Xbox One controller~~

~~With Xbox 360 controller emulation set, the host does not receive signals. Moonlight detects the controller.~~

I forgot about ViGEmBus driver on host, sorry!

**Comment by ReenigneArcher** [comment]
Xbox One controller can now be emulated in Linux, but not Windows.

**Reply by veslintu** [reply]
is xbox one controller emulation coming to windows any time soon?

**Reply by ReenigneArcher** [reply]
No

**Reply by ReenigneArcher** [reply]
There is some work being done to migrate from vigembus. https://github.com/cgutman/WinUHid

**Comment by ReenigneArcher** [comment, marked as answer]
Added in https://github.com/LizardByte/Sunshine/pull/5368 (except macOS)
